### ISSUE TITLE
Accessibility issue in the names of some symbols in the Master palette of Symbols

### ISSUE THREAD

**Issue by Daniel-R15** [musescore/MuseScore#23523]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

NVDA screen reader is used.
Creating a score in the new score window.
Adding an instrument to a new score.
Editing notes.
Opening the master palette with the key combination Shift + F9
Selecting one of the following categories with a problem:
* Accidentals;
* Grace notes;
* Noteheads;
* Layout;
* Beam properties.

Some symbols from these categories have problems with their names. Instead of the name of the symbol we hear: "Akcion icon".

[ Originally reported by user vlad-mus on musescore.org forum "Support and bug reports" on 2024-07-08:
https://musescore.org/en/node/364747#comment-1249051 ]

### Supporting files, videos and screenshots

Link to video demonstration:
Bug_report_№2_(2024-07-08_17-05-06).mp4
https://disk.yandex.ru/i/n88Vurc4RS_Y2g

### What is the latest version of MuseScore Studio where this issue is present?

MuseScore Studio, version 4.3.2

### Regression

I was unable to check

### Operating system

Windows 10 (64 bit)

### Additional context

For further confirmation of details, refer to OP vlad-mus https://musescore.org/en/user/2370413 

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
@shoogle Can you please check this on Narrator?
